projects/vdk-control-cli/src/vdk/internal/control/utils/output_printer.py

```diff
@@ -107,6 +108,40 @@ def print_dict(self, data: Dict[str, Any]) -> None:
             click.echo("{}")
 
 
+class MemoryPrinter(Printer):
```

Comment:
Maybe named it InMemoryTextPrinter 

Because the format is still in text

Reply:
yes, sure